src/app/recipes/recipe-edit.html

```diff
@@ -0,0 +1,54 @@
+<div class="recipe-detail-header" *ngIf="recipe && recipe.imageUrl"
+[ngStyle]="{ 'background-image': 'url(' + recipe.imageUrl + ')' }">
+</div>
+<div class="recipe-detail-content" *ngIf="recipe">
+<div class="page-title-header">
+  <h1>
+    {{ recipe.name }}
+    <small>Submitted by {{ recipe.userName }}</small>
+  </h1>
+</div>
+<div class="row">
+  <div class="col-12 col-sm-6 col-lg-7 col-xl-8 recipe-runway" *ngIf="recipe">
+    <div class="recipe-actions">
+    </div>
+    <div>
+        <div class="col-12 col-sm-7" *ngIf="recipe">
+            <div class="card">
+                <div class="card-body">
+                  <h1>{{ recipe.name }}</h1>
+                </div>
+                <form class="card-body" (ngSubmit)="updateRecipe()">
+                    <div class="form-group">
+                        <label for="name">Name</label>
+                        <input type="text" name="name" class="form-control" [(ngModel)]="recipe.name" />
+                    </div>
+                    <div class="form-group">
+                        <label for="category">Category</label>
+                        <input type="text" name="category" class="form-control" [(ngModel)]="recipe.category" />
+                    </div>
+                    <div class="form-group">
+                            <label for="instructions">Cooking Instructions</label>
+                            <input type="text" name="instructions" class="form-control" [(ngModel)]="recipe.instructions" />
+                    </div>
+                    <div class="form-group">
+                        <label for="instructions">Ingredients</label>
+                        <tbody>
+                            <tr *ngFor="let ingredient of recipe.ingredients">
+                                <td>{{ ingredient.name }}</td>
+                                <td><input name="ingredient" [(ngModel)]="ingredient.quantity" value="{{ingredient.quantity}}"></td>
```

Comment:
@lukiffer This value is not actually being bound to the individual ingredient. What is incorrect about this approach?

I also took a look at tacking the index and grabbing the item from the array and then pulling the quantity and that had the same result.